plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -61,10 +61,10 @@ func calculateResourceOccupancy(pod *api.Pod, node api.Node, nodeInfo *scheduler
 	cpuScore := calculateScore(totalMilliCPU, capacityMilliCPU, node.Name)
 	memoryScore := calculateScore(totalMemory, capacityMemory, node.Name)
 	glog.V(10).Infof(
-		"%v -> %v: Least Requested Priority, Absolute/Requested: (%d, %d) / (%d, %d) Score: (%d, %d)",
+		"%v -> %v: Least Requested Priority, Absolute/Requested(CPU:millicores, memory:bytes): (%d, %d) / (%d, %d) Score: (%d, %d)",
```

Comment:
Thanks David! I will modify it. One more thing, do we need to log `cpuScore` and `memoryScore` separately? Actually, we use the average value of them as the Least Requested Priority score. I thought about this but not sure to modify or not. @davidopp